dashboard/test/ui/features/teacher_tools/level_summary.feature

```diff
@@ -0,0 +1,29 @@
+@eyes
+Feature: Level summary
+
+Scenario: Free Response level 1
```

Comment:
At the beginning of eyes tests, you need a line like `When I open my eyes to test "Free Response Level Summary"`, like in [this test](https://github.com/code-dot-org/code-dot-org/blob/df510860f2f9ff48a141ff8bf138aad290093808/dashboard/test/ui/features/teacher_tools/level_types/free_response_contained_levels.feature#L9)